usr/iscsi_util.c

```diff
@@ -392,3 +397,26 @@ int iscsi_match_target(void *data, struct session_info *info)
 				     info->persistent_port, NULL,
 				     MATCH_ANY_SID);
 }
+
+/*
+ * init thread's PR_SET_IO_FLUSHER flag
+ *
+ * val: 1 to set to set thread's PR_SET_IO_FLUSHER flag
+ *      0 to clear thread's PR_SET_IO_FLUSHER flag
+ *
+ * return: return 0 on success, else error number is returned
+ */
+int init_thread_io_flusher(int val)
+{
+	if (prctl(PR_SET_IO_FLUSHER, val, 0, 0, 0) == 0)
+		return 0;
+
+	/*
+	 * prctl would return EINVAL if the kernel do not support PR_SET_IO_FLUSHER
+	 * so donot print error log if errorno is EINVAL to avoid unnecessary errorlog
+	 */
+	if (errno != EINVAL)
+		log_error("prctl could not %s thread's PR_SET_IO_FLUSHER flag due to error %s\n",
+			val ? "set" : "clear", strerror(errno));
```

Comment:
One small nit. I noticed in your patches you sometimes use formatting like above where the second line doesn't match up with the "(" above it. We generally line it up. For example if you look at the code you replaced we in iscsid.c we line it up, so you normally want to follow that.

Reply:
Yes, I was used to keep the width of a line in 100 characters. I would pay attention to avoid this when submit patches to open-iscsi.

Reply:
Since this was a "nit" (which he understands), I will resolve this one.